pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -117,11 +117,23 @@ func (s *SpdyRoundTripper) dial(req *http.Request) (net.Conn, error) {
 
 	tlsConn := tls.Client(rwc, s.tlsConfig)
 
+	// Return if we were configured to skip validation
+	if s.tlsConfig != nil && s.tlsConfig.InsecureSkipVerify {
```

Comment:
this should be after the Handshake... we still want an encrypted connection even if we don't verify the cert